### ISSUE TITLE
Details Page Prior to Version 9

### ISSUE THREAD

**Issue by ukasuviok** [cardona150/MlbTheShow18MarketAssistant#9]
This is a request to have the card details page brought back prior to release 9.

**Comment by dvt27** [comment]
Agreed. Is there any way you can bring back the details page? Really appreciate what you doing for the community
